_data/cryptoservices.yml

```diff
@@ -661,19 +633,6 @@ websites:
   bch: true
   btc: false
   othercrypto: false
-- name: Taskopus

```

Comment:
Hmmm, its gone already? I'll add all the removed entries to my list to check again later.